Gemfile

```diff
@@ -48,6 +48,8 @@ end
 
 group :test do
```

Comment:
The guard docs say to add this to the development group, maybe we add it to the group with both test and development? https://github.com/guard/guard/blob/master/README.md#installation

Reply:
D'Oh! That sounds reasonable. I'll push that up now 😅